releasenotes/notes/ignore_under_parameter-e06eab711a713571.yaml

```
@@ -0,0 +1,14 @@
+---
+features_visualization:
+  - |
+    New parameter added to the function plot_histogram. Parameter is named `ignore_under`. 
```

You can link to the method via 
```suggestion
    New parameter added to the function :func:`.plot_histogram`. Parameter is named `ignore_under`. 
```